css/main.css

```diff
@@ -0,0 +1,200 @@
+@import "../css/default.css";
+
+.container {
+  padding: 0 1rem;
+}
+
+.hero {
+  display: grid;
+  grid-template-columns: repeat(1, minmax(0, 1fr));
+  gap: 10px;
+  padding: 1rem;
+  background: var(--gray-50);
+  margin-top: 4.5rem;
+}
+
+.hero .hero-img img {
+  width: 100%;
+  object-fit: cover;
+  object-position: center;
+}
+
+.hero-content {
+  padding-top: 1rem;
+  padding-right: 1rem;
+  order: 2;
+}
+
+.hero-img {
+  order: 1;
+}
+
+.hero-description {
+  font-size: 1rem;
+  padding-top: 0.5rem;
+}
+
+.hero-btn {
+  margin: 2rem 0;
+}
+
+.category-title {
+  font-size: 1.5rem;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
```

Comment:
Improvement: It can be margin: 3rem 0 2rem 0

Reply:
Thank you. I'll refactor it.